package.json

```diff
@@ -399,19 +422,25 @@
                 "scopeName": "markdown.math_display",
                 "path": "./syntaxes/math_display.markdown.tmLanguage.json",
                 "injectTo": [
-                    "text.html.markdown"
+                    "text.html.markdown",
+                    "text.html.markdown.jsx"
                 ]
             },
             {
                 "scopeName": "markdown.math_inline",
                 "path": "./syntaxes/math_inline.markdown.tmLanguage.json",
                 "injectTo": [
-                    "text.html.markdown"
+                    "text.html.markdown",
+                    "text.html.markdown.jsx"
```

Comment:
There seems to be nothing about LaTeX in [JSX](https://reactjs.org/docs/introducing-jsx.html). Is it correct to inject them to `text.html.markdown.jsx` scope?